src/apps/explorer/components/TransanctionBatchGraph/elementsBuilder.tsx

```diff
@@ -159,9 +163,10 @@ export function buildGridLayout(
   const _nodes = nodes.map((node) => {
     const _node = {
       ...node,
-      position: {
-        y: counterRows[node.data.type],
-        x: getGridPosition(node.data.type, traders, dexes),
+      data: {
+        ...node.data,
+        row: counterRows[node.data.type],
+        col: getGridPosition(node.data.type, traders, dexes),
```

Comment:
this was causing the layout issue? nice finding!!!

Reply:
Yes, I was using the position `propesties` but this layout uses that attribute to position on the **x** and **y** axis when [defining the column/row](https://github.com/cowprotocol/explorer/blob/41-anchor-url/src/apps/explorer/components/TransanctionBatchGraph/index.tsx#L181).